docs/DeveloperGuide.md

```diff
@@ -322,7 +331,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
 * **Private contact detail**: A contact detail that is not meant to be shared with others
-
+* **t-**: Symbol for an assignment list related command
+* **p-**: Symbol for a person related command
+* **e-**: Symbol for an email related command
```

Comment:
I think we only have t- and p- commands right?

Reply:
maybe the literal meaning is wrong, glossary just used to describe something that could be awkward to the user, p-add n/NAME e/EMAIL m/MODULES, this command need to explain something related to "/n" and so on